### ISSUE TITLE
Flag by severity

### ISSUE THREAD

**Issue by silverbackdigital** [ThePatr10t/yall-Qaeda#24]
I looked through a good amount of these and some are just crowd shots and then there are a those that are threatening/malicious. It would be good if there was a way that users could indicate those.

**Comment by jantrackingnewkid** [comment]
I agree. It would also be nice to have a way to write captions/comments, especially to help track wanted suspects
